classlib/Validation.php

```diff
@@ -0,0 +1,344 @@
+<?php
+/**
+ * Validation Class
+ *
+ * @package
+ * @subpackage
+ * @category
+ * @author Sanjeeb Rao
+ */
+
+class Validation {
+    private $error = FALSE;
+    private $db_obj = NULL;
+
+    /**
+     * Initialization of Database object.
+     *
+     * @access public Constructor of Validation
+     * @param  object db_object
+     */
+    public function __construct($db_object)
+    {
+        $this->db_obj = $db_object;
+    }
+
+
+    /**
+     * To check user is valid or not.
+     *
+     * @access public  is_valid_employee
+     * @param  string email
+     * @param  string password
+     * @return mixed
+     */
+    public function is_valid_employee($email, $password = '')
+    {
+        $condition = "WHERE email = '$email' ";
+
+        if ('' !== $password)
+        {
+            $condition .= "AND password = '$password'";
+        }
+
+        $result = $this->db_obj->select('employee', ' id, is_completed ', $condition);
+
+        if (FALSE === $result)
+        {
+            $this->error = TRUE;
+            return FALSE;
+        }
+
+        $num_rows = mysqli_num_rows($result);
+
+        if (0 === $num_rows)
+        {
+            return 0;
+        }
+
+        $id = mysqli_fetch_assoc($result);
+        $this->error = TRUE;
+        return $id;
+    }
+
+
+    /**
+     * To sanitize value.
+     *
+     * @access public  sanitize_input
+     * @param  string input
+     * @return string
+     */
+    function sanitize_input($input)
+    {
+        return htmlspecialchars(stripslashes(trim($input)));
```

Comment:
use this function only while outputting to html pages
function name can be renamed for this use
stripslashes should not be required